{{ cookiecutter.repo_name }}/README.md

```diff
@@ -1,57 +1,33 @@
-    │
-    ├── docs               <- A default Sphinx project; see sphinx-doc.org for details
-    │
-    ├── models             <- Trained and serialized models, model predictions, or model summaries
-    │
-    ├── notebooks          <- Jupyter notebooks. Naming convention is a number (for ordering),
-    │                         the creator's initials, and a short `-` delimited description, e.g.
-    │                         `1.0-jqp-initial-data-exploration`.
-    │
-    ├── references         <- Data dictionaries, manuals, and all other explanatory materials.
-    │
-    ├── reports            <- Generated analysis as HTML, PDF, LaTeX, etc.
-    │   └── figures        <- Generated graphics and figures to be used in reporting
-    │
-    ├── requirements.txt   <- The requirements file for reproducing the analysis environment, e.g.
-    │                         generated with `pip freeze > requirements.txt`
-    │
-    ├── setup.py           <- makes project pip installable (pip install -e .) so src can be imported
-    ├── src                <- Source code for use in this project.
-    │   ├── __init__.py    <- Makes src a Python module
-    │   │
-    │   ├── data           <- Scripts to download or generate data
-    │   │   └── make_dataset.py
-    │   │
-    │   ├── features       <- Scripts to turn raw data into features for modeling
-    │   │   └── build_features.py
-    │   │
-    │   ├── models         <- Scripts to train models and then use trained models to make
-    │   │   │                 predictions
-    │   │   ├── predict_model.py
-    │   │   └── train_model.py
-    │   │
-    │   └── visualization  <- Scripts to create exploratory and results oriented visualizations
-    │       └── visualize.py
-    │
-    └── tox.ini            <- tox file with settings for running tox; see tox.readthedocs.io
+.
+├── data
+│   ├── external        <- Data from third party sources.
+│   ├── interim         <- Intermediate data that has been transformed.
+│   ├── processed       <- The final, canonical data sets for modeling.
+│   └── raw             <- The original, immutable data dump.
 
+├── LICENSE
+├── Makefile            <- Makefile with commands like `make data` or `make train`
+├── models              <- Trained and serialized models, model predictions, or model summaries
+├── notebooks           <- Jupyter notebooks. Naming convention is a number (for ordering),
+│                         the creator's initials, and a short `-` delimited description, e.g.
+│                         `1.0-jqp-initial-data-exploration`.
+├── README.md
+├── requirements_test.txt
+├── requirements.txt    <- The requirements file for reproducing the analysis environment, This
+│                         is automatically generated by `make update-requirements`
+├── setup.py            <- makes project pip installable (pip install -e .) so src can be imported
+├── src                 <- Source code used in the project
```

Comment:
should we name this different? <package_name> or something

Reply:
Think this is ok for now